app/src/main/java/com/github/featuredetectandroid/utils/conversions/KeypointsToOffsetList.kt

```diff
@@ -0,0 +1,6 @@
+package com.github.featuredetectandroid.utils.conversions
+
+import androidx.compose.ui.geometry.Offset
+import com.github.featuredetectlib.Keypoint
+
+fun keypointsToOffsetList(keypoints: List<Keypoint>) = keypoints.map { Offset(it.x, it.y) }
```

Comment:
This function is so short that it could be inlined. Also it's used only in one place.

Reply:
fixed